instrumentation/README.lto.md

```diff
@@ -88,16 +88,35 @@ apt-get install -y clang-12 clang-tools-12 libc++1-12 libc++-12-dev \
 ### Building llvm yourself (version 12)
 
 Building llvm from github takes quite some long time and is not painless:
-```
+```sh
 sudo apt install binutils-dev  # this is *essential*!
 git clone https://github.com/llvm/llvm-project
 cd llvm-project
 mkdir build
 cd build
-cmake -DLLVM_ENABLE_PROJECTS='clang;clang-tools-extra;compiler-rt;libclc;libcxx;libcxxabi;libunwind;lld' -DCMAKE_BUILD_TYPE=Release -DLLVM_BINUTILS_INCDIR=/usr/include/ ../llvm/
-make -j $(nproc)
-export PATH=`pwd`/bin:$PATH
-export LLVM_CONFIG=`pwd`/bin/llvm-config
+
+# Add -G Ninja if ninja-build installed
+cmake \
+    -DCLANG_INCLUDE_DOCS="OFF" \
+    -DCLANG_PLUGIN_SUPPORT="OFF" \
```

Comment:
this would break afl++ ...

Reply:
I have instrumented binaries with `afl-clang-lto` with this exact configuration on both `release/11.x` and `release/12.x` of `llvm-project`. What breakages did you observe?

Reply:
for afl-clang-lto the linker (ld.lld) loads the pass. in afl-clang-fast however clang loads the pass and disabling loading that would fail. also cmplog and laf for lto should fail because that too is done with clang

Reply:
Ah, my understanding is this build process was only for building only LTO mode, (i.e. LAF mode out of scope, hence not testing it).  Reverted in df75f62e334a081db868c5b32a73d19877a68c33